integrationTests/relayers/slowTests/framework/multiversxHandler.go

```diff
@@ -877,23 +966,24 @@ func (handler *MultiversxHandler) CreateDepositsOnMultiversxForToken(
 			createDepositGasLimit,
 			esdtTransferFunction,
 			[]string{
-				hex.EncodeToString([]byte(token.MvxChainSpecificToken)),
+				hex.EncodeToString([]byte(token.MvxUniversalToken)),
```

Comment:
SendDepositTransactionFromMultiversx and CreateDepositsOnMultiversxForToken are very similar. why do we need both?

Reply:
remained from the last refactoring. Will take out the common code, and use only one